product-service/handlers/getProductsById.ts

```diff
@@ -0,0 +1,25 @@
+import {Context, APIGatewayEvent, APIGatewayProxyResult} from "aws-lambda";
+import {getProductsByIdWithDelay} from "service/products";
+
+export async function getProductsById(
+    event: APIGatewayEvent,
+    context: Context
+): Promise<APIGatewayProxyResult> {
+    try {
+        return {
+            statusCode: 200,
+            headers: {
+                "Access-Control-Allow-Origin": "*"
+            },
```

Comment:
I think It will be better to put the repeated parts of the code into a separate helper module or response handler function.

Reply:
You are suggesting DRY over KISS.